Disaster Recovery/SafeHaven 4/Register-DataCenters-SRN-and-Peer-SRN.md

```diff
@@ -0,0 +1,128 @@
+{{{
+  "title": "Register Data Center,SRN and pair SRN's",
+  "date": "05-08-2017",
+  "author": "Mahima Kumar",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Article Overview
+
+This article focusses on how to:
+1) Register Data Centers and SRN's within the SafeHaven Console for CenturyLink Cloud, VMware vSphere and Manual(Standalone VMware ESXi hosts and DCC) datacenter type.
+2) Pair the SRN's together to set up Protection Groups.
+
+A single SafeHaven Cluster can have upto 64 datacenters registered within itself which can be accessed through a single pane of glass. Each production and recovery datacenter has to be registered within the SafeHaven Console.
+
+There are 4 **Data Center Type** types available:
+1) VMware vSphere
+2) VMware vCloud Director
+3) CenturyLink Cloud
+4) Manual (Standalone VMware ESXi hosts and DCC)
+
+**NOTE**: For CenturyLink Cloud, each sub-account has to be registered as a separate datacenter.
+
+### Requirements
+
+1) SafeHaven Cluster already installed.

```

Comment:
Change numbering from `1)` to `1.`
This will make this KB consistent with the rest of the KBs.